src/message-card/index.ts

```diff
@@ -0,0 +1,10 @@
+/*
+Copyright (c) Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+export { default as MessageCard } from './message-card';
+export * from './constants';
+export * from './styled-components';
+export * from './types';
```

Comment:
`MessageCard` uses the `secondary` and `tertiary` values from the `Button`'s `KIND` constant. Does it make sense to duplicate that constant in `MessageCard`'s `constants.ts` file and expose it? Or should devs using `MessageCard` import the `KIND` constant from `baseui/button`?

Reply:
I ended up creating a `BUTTON_KIND` constant to export from `MessageCard`. I couldn't get the docs site working otherwise.